assemblies/assembly-planner-configuration.adoc

```diff
@@ -542,6 +542,39 @@ most appenders, including the console, cause congestion with `debug` and `trace`
 Switch to an async appender to avoid this problem or turn off `debug` logging.
 ====
 
+[[monitoring]]
+[id='monitoring-the-solver-proc_{context}']
+=== Monitoring the solver
+
+{PLANNER} exposes metrics through https://micrometer.io/[Micrometer] which can be used for monitoring the solver. OptaPlanner automatically connects to configured registries when it is used in Quarkus or Spring Boot. If you use {Planner} with plain Java, you must add the metrics registry to the global registry.
+
+.Prerequisites
+* You have a plain Java OptaPlanner project.
+
+.Procedure
+. Add the following line to the global registry, where `<REGISTRY>` is the name of the registry for the monitoring system that you are using:
++
+[source,java,nowrap]
+----
+Metrics.addRegistry(new <REGISTRY>());
```

Comment:
In the community docs, this was changed to:

```
 In plain Java, add your favorite registry to the global registry:

----
MeterRegistry registry;
// Create and configure registry
// ...
Metrics.addRegistry(registry);
----

Visit the Micrometer website to see the registries available and how to configure them.
```

The main issue with this line is that many registries do not have a default constructor, meaning it will fail to compile.

Reply:
OK, I made another pass. I now want to give generic instructions followed by a specific example. Does the user create a configure in global registry, as I have shown, or are they separate steps? For exmample:
1. Create the registry for the monitoring system of your choice in Micrometer.
2. Add the registry configuration information to the global registry file.
3. dd the following line below the configuration information, where `<REGISTRY>` is the name of the registry that you configured.

Reply:
What you have now is good.